release/windows/winget/Microsoft.Sqlcmd.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.1.5 $debug=NVS1.CRLF.5-1-19041-1682.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Microsoft.Sqlcmd
+PackageVersion: 0.8.1
+InstallerLocale: en-US
```

Comment:
>stallerLocale: en-US [](http://example.com/codeflow?start=2&length=20)

when we get around to localization, can we still have just one msi that installs all the languages?